pages/blog/blog-post-template.html

```diff
@@ -1,7 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
+        <!-- MAIN CSS STYLESHEET -->
+        <link rel="stylesheet" href="/assets/css/main.css" />
+        <!-- BLOG POSTS STYLESHEET -->
         <link rel="stylesheet" href="blog-posts.css" />
+
```

Comment:
I have added our main CSS stylesheet so that our HTML components (navbar, footer, gdpr popup) can use the styling form there and we don't have to write them up again 😄